client/App.jsx

```diff
@@ -172,6 +233,9 @@ React.Component {
                         <MainImage videoEmbed={this.state.videoEmbed} onClose={this.onClose.bind(this)} previousImage={this.state.previouslySelectedImageNumber} numOfImgs={this.state.numOfImgs} onScroll={this.onClickArrow.bind(this)} onClick={this.onClickMain.bind(this)} selectedItemId={this.state.selectedItem} imgNum={this.state.imageNumber}/>
                     </div>
                 </div>
+                <div id='m_buyItemModal'>
+                    <BuyItemModal itemName={this.state.itemName} selectedItemId={this.state.selectedItem}></BuyItemModal>
+                </div>
```

Comment:
Closing tag here looks to have slid to the left

Reply:
Nah, BuyItemModal is in a separate div from the other two.